common/config/config.go

```diff
@@ -323,8 +323,18 @@ type (
 		// RPCName indicate the remote service name
 		RPCName string `yaml:"rpcName"`
 		// Address indicate the remote service address(Host:Port). Host can be DNS name.
-		// For currentCluster, it's usually the same as publicClient.hostPort
```

Comment:
Why this change? Do you run into some conflicts with master?

Reply:
I did actually, deleted it by mistake trying to fix it. I'll roll it back